src/applications/static-pages/download-1095b/components/App/index.js

```diff
@@ -60,18 +60,19 @@ export const App = ({ displayToggle, toggleLoginModal }) => {
           if (response.availableForms.length === 0) {
             recordEvent({ event: '1095b-available-forms-not-found' });
             updateFormError({ error: true, type: errorTypes.NOT_FOUND });
-          }
-          const mostRecentYearData = response.availableForms[0];
-          if (mostRecentYearData?.year) {
-            updateYear(mostRecentYearData.year);
+          } else {
+            recordEvent({ event: '1095b-available-forms-found' });
+            const mostRecentYearData = response.availableForms[0];
+            if (mostRecentYearData.year) {
+              updateYear(mostRecentYearData.year);
+            }
```

Comment:
I put all of this in the else block and also removed the safety operator. Please confirm this is functionally equivalent.